packages/components/src/components/primitives/Radio/types.ts

```diff
@@ -0,0 +1,13 @@
+import { RadioProps as ChakraRadioProps } from '@chakra-ui/react';
+
+export const radioVariants = [
+  'default',
+] as const;
+export type RadioVariant = typeof radioVariants[number];
+export const RadioSizes = ['sm', 'md', 'lg'] as const;
+export type RadioSize = typeof RadioSizes[number];
```

Comment:
Sizes according to design are `s`, `m` and `l`, being `l` the default size.

Reply:
There is not a clear guideline to follow in this case for example within the `Button` component the sizes defined are `xs`, `s` , `m` and `l`, and the current sizes defined for the button component are similar to the ones I defined in the Radio component.
Can you please share with me the right guideline?  a new ticket must be created to organize `Button` component?. 
@guilean

Reply:
Let's stick to the ones defined in figma if @tkanzakic agrees on that

Reply:
the button is not consistent with figma because the design team updated the figma file after the component was implemented, we have to update the button to match figma, figma is the source of truth